vnext/ReactUWP/Utils/BaseScriptStoreImpl.h

```diff
@@ -0,0 +1,99 @@
+#pragma once
```

Comment:
Thanks for adding this!
Did the other one (UwpPreparedScriptStore) give you issues? Seems like this one can do everything (and more) that the other one could. Maybe I'll remove UwpPreparedScriptStore after making sure this one works for us
thoughts?

Reply:
I'm not that familiar with the asynchronous programming model and the IO APIs in UWP.. and even the APPX packaging for assets etc .. I initially struggled for some time to use the UwpPrepareScriptStore and then eventually gave up :) 

I think UwpPrepareScriptStore will be valuable to stream the bytecodes from the app resources when we ship the bytecode as an application resource/asset .. I think streaming from the APPX would be a lot more performant as the entire APPX is very likely memory mapped to the process space. For dynamically generating the bytecode as we do for chakra and v8, out base implementation suffice and keeps things simple especially while we are in the experimentation phase..

Reply:
Sounds good. thank you for sharing your perspective.